geniusyield-market-maker/test/Main.hs

```diff
@@ -0,0 +1,73 @@
+module Main (main) where
+
+import           Control.Concurrent.MVar
+import           Control.Exception                         (throwIO)
+import           GeniusYield.Api.Dex.Constants             (dexInfoDefaultMainnet,
+                                                            dexInfoDefaultPreprod)
+import           GeniusYield.GYConfig
+import           GeniusYield.MarketMaker.Constants         (logNS)
+import           GeniusYield.MarketMaker.MakerBotConfig
+import           GeniusYield.MarketMaker.MakerBot          (MakerBot(..), cancelAllOrders')
+import           GeniusYield.MarketMaker.Prices
+import           GeniusYield.MarketMaker.Strategies
+import           GeniusYield.MarketMaker.Utils             (addrUser)
+import           GeniusYield.Test.MarketMaker.MakerBot
+import           GeniusYield.Test.MarketMaker.Utils
+import           GeniusYield.OrderBot.DataSource.Providers (connectDB)
+import           GeniusYield.Types                         (GYNetworkId (..), addressToText,
+                                                            gyLog', GYLog(..))
+import           GeniusYield.Types.Providers               (gyLogInfo)
+import           Test.Tasty
+import           Test.Tasty.HUnit
+
+
+-----------------------------------------------------------------------
+----------------------------- MAIN ------------------------------------
+
+runSequence :: IO Bool
+runSequence = do
+  logRef  <- newMVar []
+
+  let action           = "Run"
+  let frameworkCfgPath = "../secrets/my-atlas-config-maestro.json"
+      mBotConfigFile   = Just "../secrets/my-preprod-maker-bot-config-gens-v2-test.yaml"
+
+  coreCfg <- coreConfigIO frameworkCfgPath
+  mbc     <- readMBotConfig mBotConfigFile
+  mb      <- buildMakerBot mbc
+  di      <-
+    case cfgNetworkId coreCfg of
+      GYTestnetPreprod -> pure dexInfoDefaultPreprod
+      GYMainnet -> pure dexInfoDefaultMainnet
+      _ -> throwIO $ userError "Only Preprod and Mainnet are supported."
```

Comment:
This can be abstracted for.

Reply:
Done; removed this code duplication in `test/Main.hs`.